doc/modules/manifold.rst

```diff
@@ -248,9 +248,9 @@ The overall complexity of MLLE is
 
 .. topic:: References:
 
-   * `"MLLE: Modified Locally Linear Embedding Using Multiple Weights"
-     <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.70.382>`_
+   * :doi:`"MLLE: Modified Locally Linear Embedding Using Multiple Weights"
      Zhang, Z. & Wang, J.
+     <10.1.1.70.382>`
```

Comment:
This link gives a not found: I could not find the "real" doi for this paper. Could you please revert the modification? Or look for the right DOI.